src/BylawsText.js

```diff
@@ -0,0 +1,297 @@
+import React from 'react'
+import join from 'join-array'
+
+export default function BylawsText({ answers }) {
+  const answersArray = Object.entries(answers)
+  const anyAnswerGiven = answersArray.some(([_, answer]) => {
+    const isAnswerCheckbox = Array.isArray(answer)
+    return isAnswerCheckbox
+      ? answer.filter(Boolean).length > 0
+      : answer.length > 0
+  })
+
+  console.log('answers:', answers)
+
+  const nameAndLocationParagraph = (
```

Comment:
This seems to be a component. You might want to capitalize the first letter.

Reply:
actually I don't want it to be a component